ui-components/src/headers/Hero.tsx

```diff
@@ -46,7 +46,12 @@ const Hero = (props: HeroProps) => {
   return (
     <div className={heroClasses.join(" ")} style={styles} data-test-id="hero-component">
       <div className={innerClasses.join(" ")}>
-        <h1 className={`hero__title ${props.extraLargeTitle ? "lg:text-6.5xl" : ""}`}>
+        <h1
+          className={`hero__title 
+            ${props.extraLargeTitle ? "lg:text-6.5xl" : ""} 
+            ${props.title === t("errors.notFound.title") && "text-white"}
```

Comment:
@emilyjablonski I don't love this but I needed a way to make the title white only on the error page. LMK what you think.

Reply:
We don't want to have any business logic or translations in ui-components, because we want our components to be agnostic to the business logic. If SF or Alameda were to pull this component in, we don't know they would use this string or have this exact key in their translations file. In this case if we need to customize the h1 in a one-off case, we can add a className prop for the h1.